2021-03-03 10:13:49,760 ERROR 17543 [-/127.0.0.1/-/23ms POST /api/caUser] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'type' in 'field list'
    at Query.Sequence._packetToError (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/mysql/lib/protocol/sequences/Query.js:79:18)
    at Protocol._parsePacket (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/mysql/lib/Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (internal/streams/readable.js:309:12)
    --------------------
    at Pool.query (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/lihong/gitrep/parasitic/parasitic_api/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/ali-rds/lib/client.js:38:20)
    at RDSClient.proto.query (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/co/index.js:65:19)
    at /Users/lihong/gitrep/parasitic/parasitic_api/node_modules/co/index.js:54:5
    sql: INSERT INTO `ca`(`enroll_id`, `secret`, `name`, `type`, `remark`, `ca_id`) VALUES('admin', 'ant.design', '22a22', 'client', 'rrrr', '1')
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'type' in 'field list'"
sqlState: "42S22"
index: 0
sql: "INSERT INTO `ca`(`enroll_id`, `secret`, `name`, `type`, `remark`, `ca_id`) VALUES('admin', 'ant.design', '22a22', 'client', 'rrrr', '1')"
name: "ER_BAD_FIELD_ERRORError"
pid: 17543
hostname: lihongdeMacBook-Air.local

2021-03-03 10:17:01,669 ERROR 17609 [-/127.0.0.1/-/22ms POST /api/caUser] nodejs.TypeError: Cannot read property 'create' of undefined
    at CaUserController.create (/Users/lihong/gitrep/parasitic/parasitic_api/app/controller/base.js:23:45)
    at Object.callFn (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/koa-compose/index.js:44:32)
    at next (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/koa-compose/index.js:45:18)
    at /Users/lihong/gitrep/parasitic/parasitic_api/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/koa-compose/index.js:44:32)
    at /Users/lihong/gitrep/parasitic/parasitic_api/node_modules/koa-compose/index.js:36:12

pid: 17609
hostname: lihongdeMacBook-Air.local

2021-03-03 10:21:57,359 ERROR 17652 [-/127.0.0.1/-/23ms POST /api/caUser] nodejs.TypeError: Cannot read property 'create' of undefined
    at CaUserController.create (/Users/lihong/gitrep/parasitic/parasitic_api/app/controller/base.js:23:45)
    at Object.callFn (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/koa-compose/index.js:44:32)
    at next (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/koa-compose/index.js:45:18)
    at /Users/lihong/gitrep/parasitic/parasitic_api/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/koa-compose/index.js:44:32)
    at /Users/lihong/gitrep/parasitic/parasitic_api/node_modules/koa-compose/index.js:36:12

pid: 17652
hostname: lihongdeMacBook-Air.local

2021-03-03 10:56:03,985 ERROR 17652 [-/127.0.0.1/-/2ms POST /api/caUser] nodejs.TypeError: Cannot read property 'create' of undefined
    at CaUserController.create (/Users/lihong/gitrep/parasitic/parasitic_api/app/controller/base.js:23:45)
    at Object.callFn (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/koa-compose/index.js:44:32)
    at next (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/koa-compose/index.js:45:18)
    at /Users/lihong/gitrep/parasitic/parasitic_api/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/koa-compose/index.js:44:32)
    at /Users/lihong/gitrep/parasitic/parasitic_api/node_modules/koa-compose/index.js:36:12

pid: 17652
hostname: lihongdeMacBook-Air.local

2021-03-03 10:57:09,672 ERROR 17739 [-/127.0.0.1/-/24ms POST /api/caUser] nodejs.ER_NO_DEFAULT_FOR_FIELDError: ER_NO_DEFAULT_FOR_FIELD: Field 'attributes' doesn't have a default value
    at Query.Sequence._packetToError (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/mysql/lib/protocol/sequences/Query.js:79:18)
    at Protocol._parsePacket (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/mysql/lib/Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (internal/streams/readable.js:309:12)
    --------------------
    at Pool.query (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/lihong/gitrep/parasitic/parasitic_api/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/ali-rds/lib/client.js:38:20)
    at RDSClient.proto.query (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/co/index.js:65:19)
    at /Users/lihong/gitrep/parasitic/parasitic_api/node_modules/co/index.js:54:5
    sql: INSERT INTO `ca_user`(`enroll_id`, `secret`, `name`, `type`, `remark`, `ca_id`) VALUES('admin', 'ant.design', '22a22', 'client', 'rrrr', '1')
code: "ER_NO_DEFAULT_FOR_FIELD"
errno: 1364
sqlMessage: "Field 'attributes' doesn't have a default value"
sqlState: "HY000"
index: 0
sql: "INSERT INTO `ca_user`(`enroll_id`, `secret`, `name`, `type`, `remark`, `ca_id`) VALUES('admin', 'ant.design', '22a22', 'client', 'rrrr', '1')"
name: "ER_NO_DEFAULT_FOR_FIELDError"
pid: 17739
hostname: lihongdeMacBook-Air.local

2021-03-03 11:30:56,819 ERROR 17954 [-/127.0.0.1/-/7ms GET /api/caUser?current=1&pageSize=20&sorter=%7B%7D&filter=%7B%7D&ca_id=1] nodejs.TypeError: Cannot read property 'ca_id' of undefined
    at CaUserService.select (/Users/lihong/gitrep/parasitic/parasitic_api/app/service/caUser.js:46:64)
    at CaUserController.index (/Users/lihong/gitrep/parasitic/parasitic_api/app/controller/caUser.js:14:45)
    at Object.callFn (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/koa-compose/index.js:44:32)
    at next (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/koa-compose/index.js:45:18)
    at /Users/lihong/gitrep/parasitic/parasitic_api/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/koa-compose/index.js:44:32)

pid: 17954
hostname: lihongdeMacBook-Air.local

2021-03-03 11:33:19,342 ERROR 18074 [-/127.0.0.1/-/4ms GET /api/caUser?current=1&pageSize=20&sorter=%7B%7D&filter=%7B%7D&ca_id=1] nodejs.ReferenceError: ca is not defined
    at CaUserService.select (/Users/lihong/gitrep/parasitic/parasitic_api/app/service/caUser.js:46:61)
    at CaUserController.index (/Users/lihong/gitrep/parasitic/parasitic_api/app/controller/caUser.js:14:45)
    at Object.callFn (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/koa-compose/index.js:44:32)
    at next (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/koa-compose/index.js:45:18)
    at /Users/lihong/gitrep/parasitic/parasitic_api/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/koa-compose/index.js:44:32)

pid: 18074
hostname: lihongdeMacBook-Air.local

2021-03-03 15:00:22,662 ERROR 18088 [-/127.0.0.1/-/6ms POST /api/ca?count=5] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'createType' in 'field list'
    at Query.Sequence._packetToError (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/mysql/lib/protocol/sequences/Query.js:79:18)
    at Protocol._parsePacket (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/mysql/lib/Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (internal/streams/readable.js:309:12)
    --------------------
    at Pool.query (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/lihong/gitrep/parasitic/parasitic_api/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/ali-rds/lib/client.js:38:20)
    at RDSClient.proto.query (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/lihong/gitrep/parasitic/parasitic_api/node_modules/co/index.js:65:19)
    at /Users/lihong/gitrep/parasitic/parasitic_api/node_modules/co/index.js:54:5
    sql: INSERT INTO `ca`(`name`, `createType`, `enroll_id`, `secret`, `remark`, `extension`) VALUES('tet', 1, 'tet', 'tettet', 'tettettettettettet', ('1'))
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'createType' in 'field list'"
sqlState: "42S22"
index: 0
sql: "INSERT INTO `ca`(`name`, `createType`, `enroll_id`, `secret`, `remark`, `extension`) VALUES('tet', 1, 'tet', 'tettet', 'tettettettettettet', ('1'))"
name: "ER_BAD_FIELD_ERRORError"
pid: 18088
hostname: lihongdeMacBook-Air.local

